Documentation/ceph-upgrade.md

```diff
@@ -255,9 +272,24 @@ needed by the Operator. Also update the Custom Resource Definitions (CRDs).
 > `rbac.authorization.k8s.io/v1beta1` instead of `rbac.authorization.k8s.io/v1`
 > You will also need to apply `pre-k8s-1.16/crds.yaml` instead of `crds.yaml`.
 
+First get the latest common resources manifests that contain the latest changes for Rook v1.5.
 ```sh
-git clone --single-branch --branch v1.5.0 https://github.com/rook/rook.git
+git clone --single-branch --branch v1.5.3 https://github.com/rook/rook.git
 cd rook/cluster/examples/kubernetes/ceph
+```
+
+If you have deployed the Rook Operator or the Ceph cluster into a different namespace than
```

Comment:
This section will be useful for creating new clusters as well. How about if we add a third bullet to the end of [this section in the getting started guide](https://rook.github.io/docs/rook/v1.5/ceph-quickstart.html#deploy-the-rook-operator) so they can consider installing in a namespace other than the default `rook-ceph`? That bullet can just link to this section in the upgrade guide. Or alternatively, this section could be moved to the getting started guide and this upgrade guide could link to that. wdyt?

Reply:
Also, did you mention somewhere to also replace the `provisioner:` tags? Those are more for installation than upgrade though.

Reply:
Hmm... I think copying it to the getting started guide is better than a link.

I'll work on sed replacements for provisioner and serviceaccount tags in the quickstart guide.